### DISCUSSION TITLE
Should the default gap in the various components be removed?

### DISCUSSION THREAD

**Discussion by mikuhl-dev** [chakra-ui/panda#803] [Polls]
To me it seems weird that many of the included components come with a default gap of 10. If this is not desired you have to remember to awkwardly set the gap to 0. I feel like the name of the component should strictly follow its desired effect.

**Comment by anubra266** [comment]
What components do you mean? isn't it just stacks?

**Reply by mikuhl-dev** [reply]
Stacks and Grids from what I see, I haven't checked everything.
For example, when I think to use a Horizontal Stack, I would expect the component to stack the components horizontally, not stack the components horizontally and add a 2rem gap.

**Reply by anubra266** [reply]
@MichaelPriebe it is common to introduce gaps or spacing between stack elements to improve visual separation and readability.

**Comment by asabil** [comment]
Any updates on this? I also found the default gaps in Stacks and Grids quite surprising.
